index.html

```diff
@@ -90,25 +93,25 @@ <h2>Tonic</h2>
                 <div class="card-content">
                     <h2>Multi-Post Stories</h2>
                     <ul class="card-list1">
-                        <li>CANOPY</li>
+                        <li>FACEBOOK</li>
```

Comment:
- Good job done so far, but according to the project requirements, the whole of this section, the project section, should be rendered using Javascript. Please refactor this section to be dynamically rendered in JavaScript.
Project requirements:
`You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.`